compose.yaml

```diff
@@ -0,0 +1,121 @@
+# env variables needed
+# NGC_API_KEY
+
+services:
+  nv-embedqa-e5-v5:
+    image: "nvcr.io/nim/nvidia/nv-embedqa-e5-v5:1.0.1"
+    profiles: ["2 GPUs", "3+ GPUs"]
+    deploy:
+      resources:
+        reservations:
+          devices:
+            - driver: nvidia
+              capabilities: ["gpu"]
+              count: 1
+    ipc: host
+    environment:
+      - NGC_API_KEY=${NGC_API_KEY}
+    volumes:
+      - nim-cache:/opt/nim/.cache
+    healthcheck:
+      test: ["CMD", "python3", "-c", "import requests; resp = requests.get('http://localhost:8000/v1/health/ready'); resp.raise_for_status()"]
+      interval: 30s
+      start_period: 600s
+      timeout: 20s
+      retries: 3
+    networks:
+      - default
+  nv-rerankqa-mistral-4b-v3:
+    image: "nvcr.io/nim/nvidia/nv-rerankqa-mistral-4b-v3:1.0.2"
+    profiles: ["3+ GPUs"]
+    runtime: "nvidia"
+    deploy:
+      resources:
+        reservations:
+          devices:
+            - driver: nvidia
+              capabilities: ["gpu"]
+              count: 1
+    ipc: host
+    environment:
+      - NGC_API_KEY=${NGC_API_KEY}
+    volumes:
+      - nim-cache:/opt/nim/.cache
+    healthcheck:
+      test: ["CMD", "python3", "-c", "import requests; resp = requests.get('http://localhost:8000/v1/health/ready'); resp.raise_for_status()"]
+      interval: 30s
+      start_period: 600s
+      timeout: 20s
+      retries: 3
+    networks:
+      - default
+
+  llm-nim:
+    image: "nvcr.io/nim/meta/llama3-8b-instruct:1"
+    profiles: ["1 GPU", "2 GPUs", "3+ GPUs"]
+    deploy:
+      resources:
+        reservations:
+          devices:
+            - driver: nvidia
+              capabilities: ["gpu"]
+              count: 1
+    ipc: host
+    environment:
+      - NGC_API_KEY=${NGC_API_KEY}
+    volumes:
+      - nim-cache:/opt/nim/.cache
+    healthcheck:
+      test: ["CMD", "python", "-c", "import requests; resp = requests.get('http://localhost:8000/v1/health/ready'); resp.raise_for_status()"]
+      interval: 30s
+      start_period: 600s
+      timeout: 20s
+      retries: 3
+    networks:
+      - default
+
+  milvus:
+    image: "milvusdb/milvus:v2.4.6"
+    profiles: ["0 GPUs", "1 GPU", "2 GPUs", "3+ GPUs"]
```

Comment:
milvus should not belong to any profiles. that way it will always be started.

Reply:
The reason I added milvus and redis to all the profiles is because I still wanted to display a "0 GPU" profile. I'll take a look at this to see if I can have an "empty" profile, unless you think we don't need the 0 GPU profile to display in the drop down